Clicker/Models/ListDiffableModels/Poll.swift

```diff
@@ -17,97 +17,53 @@ enum PollState {
 
 class Poll {
     
-    let identifier = UUID().uuidString
-    var id: Int!
+    var id: Int
     var text: String
     var questionType: QuestionType
     var options: [String]
-    var results: [String:Any]
-    var state: PollState!
+    var results: [String:JSON]
```

Comment:
In the future this might be a good use of `Codable`, as a Austin presented on Sunday. This is perfectly fine for this commit though.